README.md

```diff
@@ -35,45 +35,73 @@ when building the live site).
 
 ## Install
 
-Local development requires [Vagrant](http://docs.vagrantup.com/v2/installation/).
+You can run this repo using Vagrant (preferred way) or you can run it locally.
```

Comment:
The wording of 'run this repo' seems a little odd to me. Maybe 'run the app' or 'run Viewer-Sinatra', or something like that?